ai-assistant/README.md

```diff
@@ -0,0 +1,27 @@
+# AI Assistant editor
+
+Add powerful AI content creation tools into CKEditor. Use pre-made prompts or custom queries to have AI write, rephrase, translate or summarize your text.
```

Comment:
There should probably be a comma before "or":
> Add powerful AI content creation tools into CKEditor. Use pre-made prompts or custom queries to have AI write, rephrase, translate, or summarize your text.

Reply:
Comma added.